features/local-font-access.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-10-28
-name: Local Font Access
-description: TODO
+name: Local font access
+description: The `Window.queryLocalFonts()` method provides access to an array of locally-installed fonts, each represented by a `FontData` object.
```

nits for casing and brevity

```suggestion
description: The `window.queryLocalFonts()` method returns an array of locally-installed fonts, each represented by a `FontData` object.
```